packages/schema/src/config/adhoc.ts

```diff
@@ -70,7 +70,7 @@ export default defineUntypedSchema({
    * 
    * @see  [Nuxt DevTools](https://devtools.nuxtjs.org/) for more information.
    * @experimental
-   * @type {boolean | { enabled: boolean }}
+   * @type {boolean | import('@nuxt/devtools').ModuleOptions}
```

Comment:
For this, the user might not have devtools installed yet (although we do default now to installing it with the starter)

Reply:
I've added the package to the dev dependencies, but you are right, this should either be something like

```ts
interface {
  enabled: boolean
  [key: string]: any
} 
```

or `@nuxt/devtools` should be moved to the dependencies

Reply:
We later augment the type in `.nuxt/types/schema.d.ts`:

https://github.com/nuxt/nuxt/blob/3e1ef92dc5ff808bf417ef3967365ab555036c6a/packages/nuxt/src/core/templates.ts#L124-L169

We need to make sure that whatever we set this to does not conflict with that.

I'd prefer not to install devtools as a dependency as we've kept it separate in order to iterate more quickly.

Reply:
I've removed `@nuxt/devtools` from the dev dependencies, and updated the type